unlock-app/src/pages/_app.tsx

```diff
@@ -13,7 +13,7 @@ import { ErrorFallback } from '~/components/interface/ErrorFallback'
 import { queryClient } from '~/config/queryClient'
 import { SessionProvider } from '~/hooks/useSession'
 import { ConnectModalProvider } from '~/hooks/useConnectModal'
-
+import '~/utils/bigint'
```

Comment:
added to the main app file to make sure it is available through the entire app. Maybe there is a better place to put this?

Reply:
I think that's ok